nebari-dev/nebari
# Add option to customize keycloak themes

## Reference Issues or PRs

closes #1980 



## What does this implement/fix?

_Put a `x` in the boxes that apply_


## Testing


## How to test this PR:

- Create a public github repo, containing a folder with the name to your theme (e.g demo);
- Inside such folder copy over the contents from https://github.com/keycloak/keycloak/tree/main/themes/src/main/resources-community/theme/base or use `my-cool-theme-name` example from https://github.com/keycloak/keycloak-quickstarts/tree/latest/extension/extend-admin-console.
- In your nebari-config.yaml, make sure to properly pass the URL for you github repo under the repository and branch fields, introduced in this PR
- After deploying, go to the Keycloak UI and select your new theme 
## Any other comments?
